ScorchApp/src/components/dungeonMaster/DungeonMasterMain.vue

```diff
@@ -68,6 +89,39 @@ export default {
                 response = await ItemService.getItemById(item.ItemId);
                 this.selectedItem = response.body;
             }
+        },
+        async giveItem(){
+            let success = true;
+            
+            this.successMsg = 'Successfully gave ' + this.itemQty + ' ' + this.selectedItem.Name + 
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals look at this for simpler string building